src/lxc/freezer.c

```diff
@@ -76,8 +78,9 @@ static int do_freeze_thaw(int freeze, const char *name, const char *lxcpath)
 		v[lxc_char_right_gc(v, strlen(v))] = '\0';
 
 		ret = strncmp(v, state, strlen(state));
-		if (ret != 0) {
-			lxc_monitor_send_state(name, freeze ? FROZEN : THAWED, lxcpath);
+		if (ret == 0) {
```

Comment:
...  why is inverting the check for strncmp return value and then doing the same thing valid here?  Was it wrong before?

Reply:
This looks like a comment on an old version?

Reply:
Ah no, you were right. My diff was wrong.